content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+models of the system.
+
+{{< image width="50%"
+    link="./multi-control.svg"
+    alt="A service mesh with multiple control planes"
+    title="Multiple control planes"
+    caption="A service mesh with multiple control planes"
+    >}}
+
+### Managed control plane
+
+Many cloud providers offer a control plane, which they manage on behalf of their
+customer.
+
+Typically, managed control planes guarantee some level of performance and
+availability, which you can assume when building your system. Managed control
+planes affords the following benefits:
+
+- Greatly reduce the complexity of user deployments
+- Greatly reduce system administration costs
+- Effectively eliminate the usage costs associated with the control plane:
+    - CPU
+    - Memory
+    - Network
+
+{{< image width="50%"
+    link="./managed-control.svg"
+    alt="A service mesh with a managed control plane"
+    title="Managed control plane"
+    caption="A service mesh with a managed control plane"
+    >}}
+
+## Identity and trust models
+
+When a workload instance is created within a service mesh, Istio assigns the
+workload an identity. An identity is a unique name backed by a key material, for
+example:
+
+- [X.509 certificate](https://en.wikipedia.org/wiki/X.509)
+- [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)
+
+Each identity Istio assigns is understood throughout the mesh and you can use it
+to enable mutual authentication and enforce policies.
+
+An identity contains a substring identifying the mesh that created it. For
+example in `spiffe://mytrustdomain.com/ns/default/sa/myname` the substring
+identifying the mesh is: `mytrustdomain.com`. We call this substring the {{<
+gloss>}}trust domain{{< /gloss >}} of the mesh. Every Istio mesh has a globally
+unique trust domain used to create identity names.
+
+The Certificate Authority (CA) creates the identities and their key material.
+You can verify the message sender's identity with the public key of the CA that
+created the identity. A **trust bundle** is the set of
+all CA public keys used by an Istio mesh. With a mesh's trust bundle, anyone can
+verify the sender of any message coming from that mesh.
+
+### Trust within a mesh
+
+Within a single Istio mesh, a CA creates identities and ensures that each
+workload instance has the key material needed to recognize all identities in the
+mesh. This model allows workload instances in the mesh to authenticate each
+other when communicating.
+
+{{< image width="50%"
+    link="./single-trust.svg"
+    alt="A service mesh with a certificate authority"
+    title="Trust within a mesh"
+    caption="A service mesh with a certificate authority"
+    >}}
+
+### Trust between meshes
+
+If a service in a mesh requires a service in another, you must federate identity
+and trust between the two meshes. To federate identity and trust,, you must
+exchange the trust bundles of the meshes. You can exchange the trust bundles
```

Comment:
for a multi-mesh scenario, we want to actually automate this I think. But we have nothing so far. Maybe we can mention that some of this is coming soon?

Reply:
@smawson We will iterate on the content in the future and we can add to it when the automation is in place.